src/pages/logs-dev-page.tsx

```diff
@@ -38,6 +39,9 @@ const LogsDevPage: React.FunctionComponent = () => {
     setDirectionInURL,
   } = useURLState({ attributes: availableAttributes });
 
+  const initialTenant = getInitialTenantFromNamespace(namespace);
+  const tenant = React.useRef(initialTenant);
```

Comment:
Do we need to use `useRef` here? It looks like the component will rerender every time the namespace changes (when the `ns` param changes), so I think that just setting `const tenant = getInitialTenantFromNamespace(namespace);` would achieve the same result?

Reply:
Yes you are right, thnx. I did the change